This patch will be removed when the Gutenberg demo app uses Hermes.

diff --git a/node_modules/react-native-reanimated/RNReanimated.podspec b/node_modules/react-native-reanimated/RNReanimated.podspec
index 1cbeafc..34f49db 100644
--- a/node_modules/react-native-reanimated/RNReanimated.podspec
+++ b/node_modules/react-native-reanimated/RNReanimated.podspec
@@ -80,6 +80,7 @@ Pod::Spec.new do |s|
   s.dependency 'Yoga'
   s.dependency 'DoubleConversion'
   s.dependency 'glog'
+  s.dependency 'React-jsc'
 
   if config[:react_native_minor_version] == 62
     s.dependency 'ReactCommon/callinvoker'
